apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/ProjectsResource.java

```diff
@@ -232,6 +237,32 @@ public Response getProjectMetrics(
         return Response.ok().entity(response).build();
     }
 
+    @GET
+    @Path("/feedback-scores/names")
+    @Operation(operationId = "findFeedbackScoreNamesByProjectIds", summary = "Find Feedback Score names By Project Ids", description = "Find Feedback Score names By Project Ids", responses = {
+            @ApiResponse(responseCode = "200", description = "Feedback Scores resource", content = @Content(schema = @Schema(implementation = FeedbackScoreNames.class)))
+    })
+    public Response findFeedbackScoreNames(@QueryParam("project_ids") String projectIdsQueryParam) {
+
+        var projectIds = Optional.ofNullable(projectIdsQueryParam)
+                .map(IdParamsValidator::getIds)
+                .map(List::copyOf)
```

Comment:
Minor: you don't need the defensive list copy, you can just work with the returned set.